frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -306,6 +306,9 @@ function ReaderFooter:init()
     if not self.settings.toc_markers_width then
         self.settings.toc_markers_width = DMINIBAR_TOC_MARKER_WIDTH
     end
+    if not self.settings.progress_bar_min_width then
+        self.settings.progress_bar_min_width = 20  -- unscaled_size_check: ignore
```

Comment:
So we don't have to read until `text = _("Minimal progress bar width in percentage of screen width"),` to understand it's a percentage :) either add a comment above saying `-- This is a percentage value, % of the screen width`, or rename that variable/setting to `progress_bar_min_width_pct`.